src/scheduler/scheduler.py

```diff
@@ -0,0 +1,48 @@
+from datetime import date
+
+import schedule
+
+from database.database import (
+    get_broadcast_subscribers,
+    get_raw_menu,
+    get_hidden_cuisines
+)
+from scheduler.scheduler_config import BREAKFAST_BROADCAST_TIME, DINNER_BROADCAST_TIME
+from util.const import BREAKFAST, DINNER
+from util.messages import menu_msg
+from util.util import parse_menu
+
+
+def scheduler(bot):
+    breakfast_subscribers = get_broadcast_subscribers(BREAKFAST)
+    dinner_subscribers = get_broadcast_subscribers(DINNER)
+
+    # schedule breakfast and dinner broadcasts
+    schedule.every().day\
+        .at(BREAKFAST_BROADCAST_TIME)\
+        .do(broadcast(BREAKFAST, bot, breakfast_subscribers))
+    schedule.every().day\
+        .at(DINNER_BROADCAST_TIME)\
+        .do(broadcast(DINNER, bot, dinner_subscribers))
```

Comment:
One thing to consider in the future is threading for individual jobs. Will need to check whether our local db can cope with multiple access

Reply:
Ok. In fact is there async feature in python?